web/modules/custom/asu_admin_toolbox/asu_admin_toolbox.routing.yml

```diff
@@ -0,0 +1,12 @@
+asu_admin_toolbox.solr_reindex_controller:
+  path: '/items/{node}/reindexSolr'
+  defaults:
+    _controller: '\Drupal\asu_admin_toolbox\Controller\SolrReindexController::reindexNodeRedirect'
+    _title: 'View'
+  requirements:
+    _role: 'administrator'
```

Comment:
if we provide the link to metadata mangers and admins, then we should allow them to access the route as well.

Reply:
good catch -- will update